src/App.tsx

```diff
@@ -22,7 +22,9 @@ function App() {
   const { user, loading } = useAuth();
   const { i18n } = useTranslation()
   config({ rtlEnabled: i18n.resolvedLanguage === 'ar' });
```

Comment:
You should not use multiple config method call.
Put the default currency inside the existing config method call.